### PR TITLE
Fix formatting in helm values table

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description
We have a huge table of all possible options for the helm values file for server 4. I got this in in a bit of a rush initially so this PR fixes some formatting problems.

# Reasons
https://circleci.atlassian.net/browse/DOCTEAM-723?atlOrigin=eyJpIjoiOTMyMTg1Mzg4OGI5NGFiMWE1NzI4Yzc1NjU5NTI4ZTkiLCJwIjoiaiJ9

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/server/installation/installation-reference.adoc [hunk 1]
```diff
@@ -252,145 +256,157 @@ a|
...
 |string
 |`"your-email@example.com"`
 |
 
-|kong.acme.enabled
+|`kong.acme.enabled`
 |bool
 |`false`
-|This setting will fetch and renew Let's Encrypt certs for you. It defaults to false as this only works when there's a valid DNS entry for your domain (and the app. sub domain) - so you will need to deploy with this turned off and set the DNS records first. You can then set this to true and run helm upgrade with the updated setting if you want.
+|This setting will fetch and renew Let's Encrypt certs for you. It defaults to `false` as this only works when there's a valid DNS entry for your domain (and the app. sub domain) - so you will need to deploy with this turned off and set the DNS records first. You can then set this to true and run `helm upgrade` with the updated setting if you want.
```
Comment (devinmarieb) [inline jekyll/_cci2/server/installation/installation-reference.adoc:404]: Should we link out to Let's Encrypt here? Unless it's linked out to elsewhere on the page, I'd suggest it.

File: jekyll/_cci2/server/installation/installation-reference.adoc [hunk 2]
```diff
@@ -735,32 +747,44 @@ a|
 ----
 {"bucketName":"","expireAfter":0,"gcs":{"enabled":false,"service_account":{"project_id":"... ...","type":"service_account"},"workloadIdentity":""},"s3":{"accessKey":"","enabled":false,"endpoint":"https://s3.us-east-1.amazonaws.com","irsaRole":"","secretKey":""}}
 ----
-|Object storage for build artifacts, audit logs, test results and more. One of object_storage.s3.enabled or object_storage.gcs.enabled must be true for the chart to function.
+|Object storage for build artifacts, audit logs, test results and more. One of 'object_storage.s3.enabled' or 'object_storage.gcs.enabled' must be true for the chart to function.
```
Comment (devinmarieb) [inline jekyll/_cci2/server/installation/installation-reference.adoc:750]:
```suggestion
|Object storage for build artifacts, audit logs, test results and more. One of `object_storage.s3.enabled` or `object_storage.gcs.enabled` must be true for the chart to function.
```

File: jekyll/_cci2/server/installation/installation-reference.adoc [hunk 3]
```diff
@@ -735,32 +747,44 @@ a|
...
 |string
 |`""`
-|AWS Authentication Config (3 Options). Option 1: Set accessKey and secretKey here, and CircleCI will create the secret for you. Option 2: Leave accessKey and secretKey blank, and create the secret yourself. CircleCI will assume it exists. Option 3: Leave accessKey and secretKey blank, and set the irsaRole field (IAM roles for service accounts), also set region: "your-aws-region".
+a|AWS Authentication Config. 
+
+**Option 1:** Set accessKey and secretKey here, and CircleCI will create the secret for you. 
+
+**Option 2:** Leave accessKey and secretKey blank, and create the secret yourself. CircleCI will assume it exists. 
+
+**Option 3:** Leave accessKey and secretKey blank, and set the irsaRole field (IAM roles for service accounts), also set region: "your-aws-region".
```
Comment (devinmarieb) [inline jekyll/_cci2/server/installation/installation-reference.adoc:782]: Option 3 is a little confusing. Is the irsaRole directly related to the access and secret keys? The "...and set the..." makes it seem like the two are connected, but then it's ALSO asking to set the region. Can it just be

> Leave accessKey and secretKey blank, set the irsaRole field (IAM roles for service accounts), and set region: "your-aws-region".

Should it be "your-aws-region" or `your-aws-region`?

File: jekyll/_cci2/server/installation/installation-reference.adoc [hunk 4]
```diff
@@ -1225,27 +1253,39 @@ a|
...
 |object
 |`{"project_id":"... ...","type":"service_account"}`
-|GCP Compute Authentication Config (3 Options). Option 1: Set service_account with the service account JSON (raw JSON, not a string), and CircleCI will create the secret for you. Option 2: Leave the service_account field as its default, and create the secret yourself. CircleCI will assume it exists. Option 3: Leave the service_account field as its default, and set the workloadIdentityField with a service account email to use workload identities.
+|GCP Compute Authentication Config. 
+
+**Option 1:** Set service_account with the service account JSON (raw JSON, not a string), and CircleCI will create the secret for you. 
+
+**Option 2:** Leave the service_account field as its default, and create the secret yourself. CircleCI will assume it exists. 
+
+**Option 3:** Leave the service_account field as its default, and set the workloadIdentityField with a service account email to use workload identities.
```
Comment (devinmarieb) [inline jekyll/_cci2/server/installation/installation-reference.adoc:1288]: Should these be in backticks?

```suggestion
**Option 3:** Leave the `service_account` field as its default, and set the `workloadIdentityField` with a service account email to use workload identities.
```

**Review by devinmarieb — APPROVED** [review]
@rosieyohannan I approved this so you won't have to re-request, but I left some comments and suggestions.

File: jekyll/_cci2/server/installation/installation-reference.adoc [hunk 5]
```diff
@@ -249,148 +253,160 @@ a|
...
 |string
 |`"your-email@example.com"`
 |
 
-|kong.acme.enabled
+|`kong.acme.enabled`
 |bool
 |`false`
-|This setting will fetch and renew Let's Encrypt certs for you. It defaults to false as this only works when there's a valid DNS entry for your domain (and the app. sub domain) - so you will need to deploy with this turned off and set the DNS records first. You can then set this to true and run helm upgrade with the updated setting if you want.
+|This setting will fetch and renew Let's Encrypt certs for you. It defaults to `false` as this only works when there's a valid DNS entry for your domain (and the app. sub domain) - so you will need to deploy with this turned off and set the DNS records first. You can then set this to true and run `helm upgrade` with the updated setting if you want.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server/installation/installation-reference.adoc:404]:
```suggestion
|This setting will fetch and renew link:https://letsencrypt.org/[Let's Encrypt] certs for you. It defaults to `false` as this only works when there's a valid DNS entry for your domain (and the app. sub domain) - so you will need to deploy with this turned off and set the DNS records first. You can then set this to true and run `helm upgrade` with the updated setting if you want.
```

File: jekyll/_cci2/server/installation/installation-reference.adoc [hunk 6]
```diff
@@ -719,32 +747,44 @@ a|
...
 |string
 |`""`
-|AWS Authentication Config (3 Options). Option 1: Set accessKey and secretKey here, and CircleCI will create the secret for you. Option 2: Leave accessKey and secretKey blank, and create the secret yourself. CircleCI will assume it exists. Option 3: Leave accessKey and secretKey blank, and set the irsaRole field (IAM roles for service accounts), also set region: "your-aws-region".
+a|AWS Authentication Config. 
+
+**Option 1:** Set accessKey and secretKey here, and CircleCI will create the secret for you. 
+
+**Option 2:** Leave accessKey and secretKey blank, and create the secret yourself. CircleCI will assume it exists. 
+
+**Option 3:** Leave accessKey and secretKey blank, and set the irsaRole field (IAM roles for service accounts), also set region: "your-aws-region".
```
Comment (rosieyohannan) [inline jekyll/_cci2/server/installation/installation-reference.adoc:782]:
```suggestion
**Option 3:** Leave accessKey and secretKey blank, set the irsaRole field (IAM roles for service accounts), and set region: `your-aws-region`.
```

**Pull request merged**